public/scss/authentication.scss

```diff
@@ -14,6 +14,14 @@ body.authentication {
     padding: 4rem;
   }
 
+  div.form-error {
+    margin-bottom: 1em;
```

Comment:
nitpick, but to be consistent with other Ibuildings projects (and EB/SPD) it's best to order these properties alphabetically

Reply:
Rightyouaresir!